subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -654,25 +650,68 @@ u8_t ll_adv_enable(u8_t enable)
 	ARG_UNUSED(rl_idx);
 #endif /* CONFIG_BT_CTLR_PRIVACY */
 
-	interval = adv->interval;
-	chan_map = lll->chan_map;
-	chan_cnt = util_ones_count_get(&chan_map, sizeof(chan_map));
+#if defined(CONFIG_BT_CTLR_ADV_EXT)
+	const u8_t phy = lll->phy_p;
+#else
+	const u8_t phy = 1;
```

Comment:
"Normal advertise is always at 1Mbps"

Reply:
Assuming this means suggestion to add a comment to this effect, I will do just that.